colcon_core/task/python/build.py

```diff
@@ -76,6 +76,8 @@ async def build(self, *, additional_hooks=None):  # noqa: D102
         env = dict(env)
         env['PYTHONPATH'] = str(prefix_override) + os.pathsep + \
             python_lib + os.pathsep + env.get('PYTHONPATH', '')
+        # coverage capture interferes with sitecustomize
```

Comment:
Can we link an issue or documentation in this comment which elaborates on the interplay here (perhaps even an extremely meta reference to this PR would be minimally acceptable)?

Does this affect our own coverage tests only or user-run coverage when using colcon?
If the latter, does this need a warning message to users that coverage capture will be affected?

Reply:
Added some hints in e368e5e.